rental.rb

```diff
@@ -0,0 +1,40 @@
+# class Rental
+#   attr_accessor :date, :book, :person
+
+#   def initialize(date, book, person)
+#     @date = date
+#     @book = book
+#     @person = person
+
+#     person.rentals << self if person
+#     book.rentals << self if book
+#   end
+# end
+
```

Comment:
- [ ] It would be nice if you don't leave the commented piece of code because it is against the Ruby best practices 🤔 Kindly remove this commented piece of code to make your codebase clean 👍